store/store.go

```diff
@@ -75,6 +73,8 @@ type Store interface {
 
 type GeneratedKeyStore interface {
 	Store
+	// Stats returns the current usage metrics of the key-value data store.
+	Stats() *Stats
```

Comment:
Why is this still needed? the PR readme says that this PR fixes #113 but then here you still use Stats? When are the metrics used vs Stats here?

Reply:
Stats is used for Memstore

Reply:
but is removed entirely now